### ISSUE TITLE
GitHub Action doesn't work for older pretext versions

### ISSUE THREAD

**Issue by oscarlevin** [PreTeXtBook/pretext-cli#824]
The current push action assumes a recent version of the CLI in that it calls `pretext build --deploys`.  But before that, the action installs the pretext version specified in `requirements.txt`.  This causes a failure not because the book is broken but because `requirements.txt` is out of date.

**Comment by StevenClontz** [comment]
I'm not sure how users are getting a workflow that's out of sync with their `requirements.txt`. Do you have an example story?

**Comment by StevenClontz** [comment]
In particular, I wouldn't expect a workflow generated by a version of PreTeXt-CLI to be compatible with an older (or newer) version of PreTeXt-CLI.

**Comment by oscarlevin** [comment]
It happened with dmoi.  Version 4 was on a branch and I updated the CLI there to the latest.  Still has that version when I switched back to the main branch that had the older requirements file.  But more realistic, a user upgrades pretext and doesn't update requirements.txt.  Most users I talk to don't even know they have a requirements.txt file and then freak out when they get emails saying their project is broken
